Backend/Remora.Discord.API.Abstractions/API/Voice/Gateway/Bidirectional/IVoiceSpeaking.cs

```diff
@@ -37,14 +37,9 @@ public interface IVoiceSpeaking : IVoiceGatewayCommand, IVoiceGatewayEvent
         /// </summary>
         SpeakingFlags Speaking { get; }
 
-        /// <summary>
-        /// Gets the delay of the speech event.
-        /// </summary>
-        int Delay { get; }
```

Comment:
This field is documented as being present and required in the online documentation. Can you explain why it's been removed, and where that is documented?

Reply:
This field was not present in the v4 gateway.
It seems to have been removed because Discord says they don't use it outside of the browser.

Ref: https://github.com/discord/discord-api-docs/issues/859

Reply:
Is it just not sent on the v4 gateway?

Reply:
Yes, the v4 gateway no longer sends the `Delay` field.